server.go

```diff
@@ -1,10 +1,10 @@
 package main
 
 import (
+	urlexpander "./lib"
```

Comment:
I am afraid this will break things. At least in my case I am getting
```
$ go build
server.go:4:2: local import "./lib" in non-local package
```
Seems like in Go everything is just a package and needs to by imported by it's fully qualified path to prevent `go get` and related commands from breaking.